automatic/brave/tools/helpers.ps1

```diff
@@ -0,0 +1,15 @@
+if (-not $env:ChocolateyForce) {
+  try {
+    [array]$key = Get-UninstallRegistryKey -SoftwareName 'Brave*'
+    $installedVersion = $key.Version[3..($key.Version.length - 1)]
+    $installedVersion = "$installedVersion" -replace '\s',''
+
+    if ($installedVersion -ge [Version]::Parse($env:ChocolateyPackageVersion))
+    {
+      $alreadyInstalled = $installedVersion
+      return
+    }
+  } catch {
+    # Installed version couldn't be checked, attempt installation
+  }
+}
```

Comment:
Sorry, I should have been clearer in what I meant.
I did not mean to move the code regarding acquiring the installed version inside the `if` statement, but rather to move all of the code inside this file inside a `function Your-FunctionName` instead, and then call that function from the install script.

Reply:
Is it ok?